dotnet/samples/KernelSyntaxExamples/Example57_FlowPlanner.cs

```diff
@@ -24,7 +24,7 @@
  */
 
 // ReSharper disable once InconsistentNaming
-public static class Example55_FlowPlanner
+public static class Example57_FlowPlanner
```

Comment:
In future i'd say don't worry about renaming to make rebasing easier and when we go to main eventually we can make the change once to match what's needed.